keyboards/binepad/bn009r2/keymaps/via/keymap.c

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 Binepad
```

Comment:
Ideally this could also be converted to json, and rules.mk removed. See [example](https://github.com/qmk/qmk_firmware/pull/19967#discussion_r1120980854).